params/index.yaml

```diff
@@ -0,0 +1,9 @@
+StateId:
+  name: state_id
+  in: path
+  required: true
+  schema:
+    type: string
+  description: |
+    State identifier.
+    Can be one of: "head" (canonical head in node's view), "genesis", "finalized", "justified", \<slot\>, \<stateRoot\>.
```

Comment:
Slot and state root were going to be `/slot/<number>` and `/state_root/<hex>` to avoid duck typing the variable in the path, right? Or does that not fit in the API definition? Should we change it again?

Reply:
Sorry, forgot about that. Ill update with those, it will dsplay better in swagger anyways